apps/cowswap-frontend/public/emergency.js

```diff
@@ -7,6 +7,41 @@ if (window.location.host === 'barn.cowswap.exchange') {
   window.location.href = 'https://barn.cow.fi'
 }
 
+/**
+ * Remove old versions of the local storage atom stores
+ * We rely on the fact that store names are in the format {name}Atom:v{version}
+ * Since outdated versions of the stores are not used anymore, we should remove them to not exceed the storage limit
+ */
+;(function () {
+  const atomStoreRegex = /Atom\:v\d/
+
+  // Find the latest version of each store
+  const storePerVersion = Object.keys(localStorage)
+    // Take only the atom stores with versions
+    .filter((key) => key.match(atomStoreRegex))
+    .reduce((acc, key) => {
+      const [name, version] = key.split(':v')
```

Comment:
This seems a bit fragile

Reply:
`"cow:veal:v1".split(':v')`

`(3) ['cow', 'eal', '1']`

Reply:
Why not using a REGEX?

Reply:
>Why not using a REGEX?

I love Regexes :) But in this context I feel more safe using code

Reply:
P.s. we also have `.filter((key) => key.match(atomStoreRegex))` line which protects us from irrelevant values